### PR TITLE
Support Decimal fields while deserializing.

### PR DESCRIPTION
**Pull request by jensens**
DX Decimal fields were not support until now.

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: news/903.bugfix [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Fix mssing `Decimal` field deserializer.
```
Comment (davisagli) [inline news/903.bugfix:1]:
```suggestion
Fix missing `Decimal` field deserializer.
```

**Review by davisagli — COMMENTED** [review]
We should add docs about decimals to https://6.docs.plone.org/plone.restapi/docs/source/usage/serialization.html and mention that they should be represented as strings rather than floats.

**Comment by jensens** [comment]
> We should add docs about decimals to https://6.docs.plone.org/plone.restapi/docs/source/usage/serialization.html and mention that they should be represented as strings rather than floats.

Done

File: docs/source/usage/serialization.md [hunk 2]
```diff
@@ -31,16 +31,28 @@ Since JSON does not have native support for dates and times, the Python and Zope
 | `DateTime("2015/11/23 19:45:55")`    | `"2015-11-23T19:45:55"` |
 
 
+## Decimal Type
+
+The [Python Decimal type](https://docs.python.org/3/library/decimal.html) supports correctly rounded decimal floating point arithmetic.
+To keep the precision serialization results in a string.
```
Comment (davisagli) [inline docs/source/usage/serialization.md:37]:
```suggestion
To keep the precision, serializing the value to JSON results in a string.
```

**Review by davisagli — APPROVED** [review]
One small suggestion to make a sentence in the docs easier to read, otherwise lgtm

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by mohammadtaranom** [comment]
Hello. Where can I download penetration testing tools?

**Comment by jensens** [comment]
> Hello. Where can I download penetration testing tools?

This is OT here, esp on this thread. Go to https://community/plone/org for this kind of question.

**Pull request merged**